Jellyfin.Data/Entities/AudioWaveformInfo.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Jellyfin.Data.Entities;
+
+/// <summary>
+/// An entity representing the metadata for a group of trickplay tiles.
+/// </summary>
+public class AudioWaveformInfo
+{
+    /// <summary>
+    /// Gets or sets the ID of the associated item.
+    /// </summary>
+    /// <remarks>
+    /// Required. Acts as the primary key.
+    /// </remarks>
+    [Key]
+    [JsonIgnore]
+    public Guid ItemId { get; set; }
```

Comment:
You are missing the navigation property

```
public BaseItemEntity Item { get; set; }
```

if not added efcore does not know this is a forgin key